README.md

```diff
@@ -296,6 +296,27 @@ In the event that the EigenDA disperser or network is down, the proxy will retur
 
 This behavior is turned on by default, but configurable via the `--eigenda.confirmation-timeout` flag (set to 15 mins by default currently). If a blob is not confirmed within this time, the proxy will return a 503 status code. This should be set long enough to accomodate for the disperser's batching interval (typically 10 minutes), signature gathering, and onchain submission.
 
+#### CORS Support <!-- omit from toc -->
+The proxy can be configured to support Cross-Origin Resource Sharing (CORS), enabling web applications to make requests to the proxy server from different origins. This is useful for JavaScript clients or web applications that need to interact with the proxy directly from the browser.
+
+To enable CORS, use the following CLI flags:
+- `--cors`: Enable CORS for all origins (allows cross-origin requests)
+- `--cors-allowed-domains`: A comma-separated list of domains to allow CORS from (e.g., `google.com,localhost:4157`). If this is empty and `--cors` is set, all origins will be allowed.
+
+Example usage:
+```bash
+# Allow CORS from any origin
+./bin/eigenda-proxy --cors
+
+# Allow CORS only from specific domains
+./bin/eigenda-proxy --cors --cors-allowed-domains="myapp.com,localhost:3000"
+```
```

Comment:
why not use a single flag `--cors--allowed-domains` which requires explicitly setting to * to allow all domains? Seems simpler UX and also safer (won't turn on CORS for all origins by mistake)?